pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+## EmberJS Open Meeting Notes
+
+
+### Meeting Info
+
+<!-- Replace the ALL CAPS information with the relevant data for the meeting -->
+- Meeting: EVENT NAME
+- Date: DATE
+- Facilitator: NAME (GITHUB USERNAME)
+
+### Agenda
+
+Agendas have two sections: 
+
+1. Standing items: should happen every meeting
+2. Additional discussion: for any other topic to be discussed
+
+#### Standing Item: General Admin
+
+<!-- Facilitator notes: -->
+<!-- Ask these questions and document if there are responses -->
+<!-- If any of these questions have a YES response, please also post to the core meta channel in Discord. -->
+
+- Any new, known blockers for our users in ember-source?
+- Any show-stopping [issues](https://github.com/emberjs/ember.js/issues) that need more immediate attention?
+- Any [pull requests](https://github.com/emberjs/ember.js/pulls) that needs more urgent review?
+
+#### Standing Item: RFC Review
+
+<!-- Facilitator notes: 
+
+Review each type of RFC PR and determine if there are any **NEW** comments or feedback.
+Document discussion (the general gist) in these notes.
+Try to document concerns raised in the RFC PR itself.
+
+Important note: it may not be possible to get through each section in a single meeting, that's fine!
+You can determine as the facilitator what is most useful for that meeting.
+
+-->
+
+TODO: 
+- [ ] Review new comments or feedback in the RFC, and the checklist in each PR description.
+- [ ] Determine if there are any new actions to take, or if any checkboxes have been completed.
+- [ ] Update this document with notes.
```

Comment:
It sounds like this template pertains only to the RFC review meetings. I think we had said that we wouldn't be taking notes for those meetings but would record outcomes on the particular RFCs. Am I remembering that correctly?

Reply:
That seems fair but also then makes me wonder if there's a point to having notes at all if we're not going to write down what was reviewed/discussed? 

WDYT?